django-s3-file-field-client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "django-s3-file-field-client",
+  "version": "0.0.1",
+  "description": "",
+  "author": "Girder",
+  "license": "MIT",
+  "main": "build/client.js",
+  "module": "build/client.js",
+  "types": "build/client.d.ts",
+  "files": [
+    "build",
+    "src"
+  ],
+  "scripts": {
+    "clean": "rimraf build",
+    "compile": "tsc -p tsconfig.lib.json",
+    "lint": "eslint src/**.ts",
+    "build": "npm run clean && npm run compile && webpack --mode=production",
+    "prepare": "echo 'dummy prepare since prepack has no dev dependencies'",
+    "prepack": "npm run build"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
+    "css-loader": "^3.5.3",
+    "eslint": "^7.2.0",
+    "json-loader": "^0.5.7",
+    "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^4.14.1",
+    "rimraf": "^3.0.2",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.2.1",
+    "ts-loader": "^7.0.5",
+    "typescript": "^3.9.5",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11"
+  },
+  "dependencies": {
+    "axios": "^0.20.0"
```

Comment:
At some point, I'd like to see if we can drop this and use `fetch`, but that's obviously outside of scope here.